suite-native/module-device-settings/src/components/DeviceBluetoothCard.tsx

```diff
@@ -0,0 +1,72 @@
+import { useCallback } from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { useAlert } from '@suite-native/alerts';
+import { Button, Text, VStack } from '@suite-native/atoms';
+import { useBluetoothDevice, useBluetoothSettings } from '@suite-native/bluetooth';
+import { Translation } from '@suite-native/intl';
+import {
+    DeviceSettingsStackParamList,
+    DeviceStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { SettingsCardWithIconLayout } from '@suite-native/settings';
+import { useToast } from '@suite-native/toasts';
+
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceStackRoutes.DeviceSettings
+>;
+
+export const DeviceBluetoothCard = () => {
+    const { showAlert } = useAlert();
+    const { showToast } = useToast();
+    const navigation = useNavigation<NavigationProp>();
+
+    const { unpairBluetoothDevice } = useBluetoothDevice();
+    const { openBluetoothSettings } = useBluetoothSettings();
+
+    const unpairTrezor = useCallback(async () => {
```

Comment:
what is the benefit of this useCallback?

```suggestion
    const unpairTrezor = useCallback(async () => {
```

Reply:
A leftover I guess, removed.